package.json

```diff
@@ -28,6 +28,7 @@
     "ethers": "^5.7.2"
   },
   "devDependencies": {
+    "@cosmjs/cosmwasm-stargate": "^0.31.1",
```

Comment:
this should be a normal dependency, since we'll export the deployment scripts from this package

Reply:
moved it to normal dependencies, I put it in dev since there were other dependencies used in the script that were in dev as well, should those be moved as well?